app/models/spotlight/resources/iiif_manifest/metadata.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+
+module Spotlight
+  module Resources
+    ###
+    #  A simple class to map the metadata field
+    #  in a IIIF document to label/value pairs
+    #  This is intended to be overriden by an
+    #  application if a different metadata
+    #  strucure is used by the consumer
+    #  override with Spotlight::Engine.config.iiif_metadata_class
+    class IiifManifest::Metadata
```

Comment:
Instead of disabling the `Style/ClassAndModuleChildren` cop, would it make sense to move this class up a level and rename it `IiifMetadata` or `IiifManifestMetadata`? It should be okay to rename this class as long as you also change the default configuration for the class that's set in engine.rb. What do you think?